beets/dbcore/db.py

```diff
@@ -342,15 +342,17 @@ def __delattr__(self, key):
 
     # Database interaction (CRUD methods).
 
-    def store(self):
+    def store(self, fields_to_store=None):
         """Save the object's metadata into the library database.
         """
         self._check_db()
 
         # Build assignments for query.
         assignments = []
         subvars = []
-        for key in self._fields:
+        if fields_to_store is None:
+            fields_to_store = self._fields
```

Comment:
This is again just cosmetic, but I like to put these default-value `if` blocks to the very top of the function to make it clear what the default values are.